src/test/java/de/tum/in/www1/artemis/participation/SubmissionIntegrationTest.java

```diff
@@ -171,7 +175,7 @@ void testGetSubmissionsOnPageWithSize() throws Exception {
     @Test
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void testGetSubmissionsOnPageWithSize_exerciseNotFound() throws Exception {
-        long randomExerciseId = 12345L;
+        long randomExerciseId = UUID.randomUUID().getMostSignificantBits();
```

Comment:
this test was also flaky for me and I fixed it.

Reply:
Can you still use a fixed ID to avoid making the test non-deterministic?